src/clj/maksut/payment/payment_service.clj

```diff
@@ -36,6 +39,11 @@
     (str/ends-with? order-id "-1") (get-translation (keyword language-code) :kuitti/käsittely)
     (str/ends-with? order-id "-2") (get-translation (keyword language-code) :kuitti/päätös)))
 
+(defn- create-receipt-description [language-code order-id]
+  (cond
+    (str/ends-with? order-id "-1") (get-translation (keyword language-code) :kuitti/käsittely-lr)
```

Comment:
Tätä endswith päättelyä on aika paljon, pitäisikö tehdä funktiot:

order-is-in-kasittely/processing
order-is-in-paatos/jotain